drivers/can/Kconfig.native_linux

```diff
@@ -0,0 +1,37 @@
+# Native Linux SocketCAN configuration options
+
+# Copyright (c) 2022 Martin Jäger <martin@libre.solar>
+# SPDX-License-Identifier: Apache-2.0
+
+config CAN_NATIVE_LINUX
+	bool "Native Linux SocketCAN Driver"
+	default y
+	depends on DT_HAS_ZEPHYR_NATIVE_LINUX_CAN_ENABLED
+	help
+	  Enable native Linux SocketCAN Driver
+
+if CAN_NATIVE_LINUX
+
+config CAN_NATIVE_LINUX_RX_THREAD_STACK_SIZE
```

Comment:
in the POSIX arch you are not using the Zephyr defined stacks like in the real embedded targets. You can just use ARCH_POSIX_RECOMMENDED_STACK_SIZE